File: CS/Solution1.Module/MyDomainObject.cs
using System;

using DevExpress.Xpo;

using DevExpress.ExpressApp;
using DevExpress.Persistent.Base;
using DevExpress.Persistent.BaseImpl;
using DevExpress.Persistent.Validation;

namespace Solution1.Module {
    [DefaultClassOptions]
    public class MyDomainObject : BaseObject {
        public MyDomainObject( Session session )
            : base( session ) {
        }
        private string field1;
        public  string Field1 {
            set {
                SetPropertyValue("Field1", ref field1, value);
            }
            get {
                return field1;
            }
        }
        private string field2;
        public string Field2 {
            set {
                SetPropertyValue( "Field2", ref field2, value );
            }
            get {
                return field2;
            }
        }
        private string field3;
        public string Field3 {
            set {
                SetPropertyValue( "Field3", ref field3, value );
            }
            get {
                return field3;
            }
        }
        private string field4;
        public string Field4 {
            set {
                SetPropertyValue( "Field4", ref field4, value );
            }
            get {
                return field4;
            }
        }
        private string field5;
        public string Field5 {
            set {
                SetPropertyValue( "Field5", ref field5, value );
            }
            get {
                return field5;
            }
        }
    }

}

File: VB/Solution1.Module/MyDomainObject.vb
Imports Microsoft.VisualBasic
Imports System

Imports DevExpress.Xpo

Imports DevExpress.ExpressApp
Imports DevExpress.Persistent.Base
Imports DevExpress.Persistent.BaseImpl
Imports DevExpress.Persistent.Validation

Namespace Solution1.Module
	<DefaultClassOptions> _
	Public Class MyDomainObject
		Inherits BaseObject
		Public Sub New(ByVal session As Session)
			MyBase.New(session)
		End Sub
		Private field1_Renamed As String
		Public Property Field1() As String
			Set(ByVal value As String)
				SetPropertyValue("Field1", field1_Renamed, value)
			End Set
			Get
				Return field1_Renamed
			End Get
		End Property
		Private field2_Renamed As String
		Public Property Field2() As String
			Set(ByVal value As String)
				SetPropertyValue("Field2", field2_Renamed, value)
			End Set
			Get
				Return field2_Renamed
			End Get
		End Property
		Private field3_Renamed As String
		Public Property Field3() As String
			Set(ByVal value As String)
				SetPropertyValue("Field3", field3_Renamed, value)
			End Set
			Get
				Return field3_Renamed
			End Get
		End Property
		Private field4_Renamed As String
		Public Property Field4() As String
			Set(ByVal value As String)
				SetPropertyValue("Field4", field4_Renamed, value)
			End Set
			Get
				Return field4_Renamed
			End Get
		End Property
		Private field5_Renamed As String
		Public Property Field5() As String
			Set(ByVal value As String)
				SetPropertyValue("Field5", field5_Renamed, value)
			End Set
			Get
				Return field5_Renamed
			End Get
		End Property
	End Class

End Namespace
